task/deprecated-image-check/0.1/deprecated-image-check.yaml

```
@@ -33,11 +35,18 @@ spec:
       image: registry.access.redhat.com/ubi8/ubi
       script: |
         #!/usr/bin/env bash
-        echo "Querying Pyxis for $(params.IMAGE_REPOSITORY)..."
-        http_code=$(curl -s -k -o $(workspaces.sanity-ws.path)/repository_data.json -w '%{http_code}' "https://catalog.redhat.com/api/containers/v1/repositories/registry/$(params.IMAGE_REGISTRY)/repository/$(params.IMAGE_REPOSITORY)")
+        IFS=$'\n' read -d'' -r -a IMAGE_ARRAY < <(echo -n $BASE_IMAGES_DIGESTS | sed 's/\\n/\'$'\n''/g' && printf '\0')
```

Have you tried code? I feel it should be like below
```suggestion
BASE_IMAGES_DIGESTS=$(echo $(params.BASE_IMAGES_DIGESTS) | sed 's/\\n/\'$'\n''/g' && printf '\0')
IFS=' ' read -a IMAGE_ARRAY <<< $BASE_IMAGES_DIGESTS
```

---

It's probably because your shell version is higher.  within registry.access.redhat.com/ubi8/ubi should work fine.